config.go

```diff
@@ -25,7 +33,9 @@ var (
 		},
 		[]string{"driver", "host", "database", "user", "sql_job", "query"},
 	)
-	reEnvironmentPlaceholders = regexp.MustCompile(`{{.+?}}`)
+	tmplStart                 = getenv("TEMPLATE_START", "{{")
```

Comment:
We don't have a great place to document this yet, but could you add it [to the example file](https://github.com/justwatchcom/sql_exporter/blob/master/examples/kubernetes/configmap.yml#L13) and add a comment there explaining that the template variables can be set through this environment variable? Thanks!